app.rb

```diff
@@ -5,4 +5,18 @@
 
 get '/cars' do
   @cars = Car.all
-end
\ No newline at end of file
+  erb :index
+end
+
+get 'cars/:id' do
+  @cars = Car.find(params[:id])
+end
+
+get '/cars/new' do
+  erb :index
+end
+
+post '/cars' do
+  @cars = Car.create(params[:id])
```

Comment:
You need to pass in the properties of the new car that are submitted through the form. Passing in `params[:id]` here doesn't do anything, especially because the car has not been created yet (and thus doesn't have an id).